src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -106,10 +106,25 @@ HRESULT OpenTerminalHere::GetState(IShellItemArray* /*psiItemArray*/,
     // E_PENDING and this object will be called back on a background thread with
     // fOkToBeSlow == TRUE
 
-    // We however don't need to bother with any of that, so we'll just return
-    // ECS_ENABLED.
+    // We however don't need to bother with any of that.
+
+    // If no item was selected when the context menu was opened and Explorer
+    // is not at a valid path (e.g. This PC or Quick Access), we should hide
+    // the verb from the context menu.
+    if (psiItemArray == nullptr)
+    {
+        const auto path = this->_GetPathFromExplorer();
+        *pCmdState = path.empty() ? ECS_HIDDEN : ECS_ENABLED;
+    }
+    else
+    {
+        winrt::com_ptr<IShellItem> psi;
+        psiItemArray->GetItemAt(0, psi.put());
+        SFGAOF attributes;

```

Comment:
You can add `SFGAOF` to `.github/actions/spelling/expect/expect.txt` to fix this spelling warning.

Reply:
Done!